templates/barbican/config/00-default.conf

```diff
@@ -21,13 +21,13 @@ interface = internal
 {{ end }}
 
 [keystone_notifications]
-enable = false
+enable = true
 topic = notifications
```

Comment:
I thought we decided to use topic = barbican_notifications so that barbican would get its own listening queue